### ISSUE TITLE
Table size limit and scrolling

### ISSUE THREAD

**Issue by darenwelsh** [enterprisemediawiki/SemanticActions#7]
- [x] Limit the height of the table
- [x] Make columns scroll if they exceed this height
- [x] Increase query results limit
- [x] How do we handle when "further results" is triggered (>limit results)?

**Comment by darenwelsh** [comment]
![image](https://user-images.githubusercontent.com/6728676/62652216-72586b00-b920-11e9-8a08-9d384d537fa7.png)

**Comment by darenwelsh** [comment]
```
table.actionable_board_action {
	display: block;
	height: 700px;
	overflow-y: scroll;
}
```

This added to the CSS will make the columns scroll. Ideally the height would be set by browser height, but I'm not sure how to do that with CSS (and not JS).

**Comment by darenwelsh** [comment]
This is closed for now by e3e7c21e98360d4ba5feb492eeffb9ad70d09bb0. It can certainly be improved.
